addons/web/static/src/legacy/scss/bootstrap_overridden.scss

```diff
@@ -250,21 +250,10 @@ $kbd-bg: $o-gray-100 !default;
 $kbd-box-shadow: 0px 1px 1px rgba($o-black, 0.2), inset 0px -1px 1px 1px rgba($o-gray-200, 0.8), inset 0px 2px 0px 0px rgba($o-white, 0.8) !default;
 
 // Input
-
 $input-bg: $o-white !default;
-$form-check-input-border-radius: 0 !default;
-$form-check-input-checked-color: $o-brand-lightsecondary !default;
-$form-check-input-checked-border-color: $o-brand-primary !default;
 $form-check-input-checked-bg-color: $o-brand-primary !default;
-
-$form-switch-color: rgba($black, .25) !default;
-$form-switch-checked-color: $o-brand-primary !default;
-$form-switch-circle-bg-color: $input-bg !default;
-$form-switch-circle-checked-bg-color: $input-bg !default;
-$form-switch-icon-color: $form-switch-color !default;
-$form-switch-icon-checked-color: $form-switch-checked-color !default;
-$form-switch-bg-image: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'><circle r='3' fill='#{$form-switch-circle-bg-color}'/><line x1='-1.5' y1='-1.5' x2='1.5' y2='1.5' stroke='#{$form-switch-icon-color}' stroke-width='0.75'/><line x1='1.5' y1='-1.5' x2='-1.5' y2='1.5' stroke='#{$form-switch-icon-color}' stroke-width='0.75'/></svg>") !default;
-$form-switch-checked-bg-image: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'><circle r='3' fill='#{$form-switch-circle-checked-bg-color}'/><path d='m-2.125,-0.5l1.75,1.75l2.5,-2.5' fill='none' stroke='#{$form-switch-icon-checked-color}' stroke-width='0.75'/></svg>") !default;
+$form-switch-checked-color: $input-bg !default;
+$form-switch-checked-bg-image: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'><circle r='3' fill='#{$form-switch-checked-color}'/></svg>") !default;
```

Comment:
`$form-switch-checked-bg-image :  [...] fill='#{$form-switch-checked-color}'`.... 
should we use `form-switch-circle-checked-bg-color` variable instead ?
I think we're missing something here :/